.gitignore

```diff
@@ -18,6 +18,9 @@ hs_err_pid[0-9]*.log
 # Test sandbox files
 src/test/data/sandbox/
 
+# Documentation markdown local test files
+docs/Gemfile.lock
+
```

Comment:
What is the reason for removing the Gem file? According to this page [https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore](https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore), `Gemfile.lock` should not be gitignored.

Reply:
It was because the file has changed after I set up Jekyll local test. Thanks for the reminder and I'll add it back in.